game_patch/misc/misc.cpp

```diff
@@ -384,7 +384,7 @@ CodeInjection game_set_file_paths_injection{
         if (rf::mod_param.found()) {
             std::string mod_dir = "mods\\";
             mod_dir += rf::mod_param.get_arg();
-            rf::file_add_path(mod_dir.c_str(), ".bik", false);
+            rf::file_add_path(mod_dir.c_str(), ".bik" "bluebeard.bty", false);
```

Comment:
Are you sure it works? Not sure what you intended with this code but AFAIK `".bik" "bluebeard.bty"` is treated by C++ compiler exactly the same as `".bikbluebeard.bty"` (strings are merged together). You probably meant `".bik bluebeard.bty"`. This could work if I understand the code correctly, but it's not how this function should be called. Second parameter of `file_add_path` is a list of extensions separated by space. You are not supposed to put full filenames there. Please change it to `".bik .bty"`

Reply:
I have tested it and it did work, but your suggestion does make a lot more sense. There will never be another `.bty` except `bluebeard.bty`, but that doesn't really matter - there's no downside and the code is just more simple this way. Will make this adjustment, thanks!